chain/ethereum/src/runtime/runtime_adapter.rs

```diff
@@ -132,6 +147,35 @@ fn ethereum_call(
     }
 }
 
+fn eth_get_balance(
+    eth_adapter: &EthereumAdapter,
+    ctx: HostFnCtx<'_>,
+    wasm_ptr: u32,
+) -> Result<AscPtr<AscBigInt>, HostExportError> {
```

Comment:
Do we need to charge gas for this call? I am not sure about the mechanics/reasoning of gas, but it feels wrong to make this free.

Reply:
Right, this shouldn't be gas free. @leoyvens how would we determine a good gas value for `eth_getBalance`?

Reply:
I suppose we can start with charging the same we charge for `eth_call`.